### ISSUE TITLE
Panic when file not exist

### ISSUE THREAD

**Issue by bungcip** [NishanthSpShetty/crust#1]
Running `crust` and give it some not existing filename will trigger a panic
```
D:\repo\crust>cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `target\debug\crust.exe`
Enter the C/C++ file to be converted to Rust : dsfdf
Enter the translation mode [(S/s)trict/(L/l)oose] :
Do you want to create a cargo project :[Y/N]
thread 'main' panicked at 'Unable to open input source file.', src\main.rs:56
note: Run with `RUST_BACKTRACE=1` for a backtrace.
error: process didn't exit successfully: `target\debug\crust.exe` (exit code: 101)
```

**Comment by NishanthSpShetty** [comment]
Crust calles panic!() on open fail.
Fixed.

**Comment by nikhilsaldanha** [comment]
Thanks for pointing that out. It's fixed! Check commit 0b48025
